Mobile/Android/Templates/xxx-clearing_account_info_passcode.md

```diff
@@ -0,0 +1,17 @@
+###  Bug: Clearing app data removes passcode 
+
+#### Pr: https://github.com/owncloud/android/pull/XXX 
+
+Devices:
+
```

Comment:
It will be tested as required. Results will be included in "Devices" and "Result" sections when the tests will be performed, but not in test plan.